roles/install_openstack_ca/tasks/main.yml

```diff
@@ -14,18 +14,25 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Get CA bundle data with retries
-  environment:
-    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
-    PATH: "{{ cifmw_path }}"
-  ansible.builtin.command:
-    cmd: 'oc get secret combined-ca-bundle -n openstack -o "jsonpath={.data.tls-ca-bundle\.pem}"'
-  retries: 10
-  no_log: true
-  delay: 3
-  until: ca_bundle_data.rc == 0
-  register: ca_bundle_data
-  ignore_errors: true
+# Here we rescue and not force error to preseve
```

Comment:
I'm not sure of the scenario where we want to ignore failures here, if someone knows, let me know and I will add a comment.